contrib/cirrus/setup_environment.sh

```diff
@@ -130,25 +115,14 @@ if ((CONTAINER==0)); then  # Not yet running inside a container
 fi
 
 # Which distribution are we testing on.
-case "$OS_RELEASE_ID" in
-    debian)
-        showrun echo "more conditional setup for debian"
-        # FIXME 2023-04-11: workaround for runc regression causing failure
-        # in system tests: "skipping device /dev/char/10:200 for systemd"
-        # (Checked on 2023-08-08 and it's still too old: 1.1.5)
-        # FIXME: please remove this once runc >= 1.2 makes it into debian.
-        showrun modprobe tun
-        ;;
-    fedora)
-        showrun echo "conditional setup for fedora"
-        if ((CONTAINER==0)); then
-            # All SELinux distros need this for systemd-in-a-container
-            msg "Enabling container_manage_cgroup"
-            showrun setsebool container_manage_cgroup true
-        fi
-        ;;
-    *) die_unknown OS_RELEASE_ID
-esac
+if [[ "$OS_RELEASE_ID" == "fedora" ]]; then
```

Comment:
should we keep the old case statement and juts NOP out the debian case instead? I am sure sooner or later we will again need some special magic on debian and then have to change it back. It also makes it clear what distros we are testing.

Reply:
Sounds reasonable and potentially helpful, thanks for the suggestion.  I'll stage that change for my next push.